gulpfile.js

```diff
@@ -152,18 +149,12 @@ gulp.task('prod', function (cb) {
     runSequence('build-prod', 'server', cb);
 });
 
-gulp.task('archive-dev', ['build-dev'], function () {
-    return gulp.src(['.dist/dev/**'])
+// Note there's a race condition if we have build-dev & build-prod run as dependent tasks of archive.
+// those need to be run as seperate steps.
```

Comment:
I was seeing similiar stuff yesterday, and I think I tried that and it didn't work.  I was getting errors that a done callback was called more than once.  I think it has something to do with the fact that both prod build & dev build use the same task, but flip a global RELEASE variable to do it.   At any rate, what I have here now works & I've tested it out.